hilo.rb

```diff
@@ -1,2 +1,22 @@
-number = rand(1..100)
-puts number 
\ No newline at end of file
+random_number = rand(1..100)
+puts "Enter a number between 1 and 100. press enter to submit your guess"
+input = gets.chomp
+guess = input.to_i
```

Comment:
.to_i seems to return 0 if it cannot parse the string it's passed so is that enough of a check for user error becasue I already have an if statement below that catches guesses that are less than 1? I have also added to the error message to cater for this case (see line 13)

Reply:
I can see you have opted to use `.to_i`.  Marking this a resolved.